qemu/tests/cfg/blockdev_snapshot_data_file.cfg

```diff
@@ -0,0 +1,30 @@
+- blockdev_snapshot_data_file:
+    type = blockdev_snapshot_data_file
+    virt_test_type = qemu
+    images += " data"
+    force_create_image_data = yes
+    force_remove_image_data = yes
+    start_vm = yes
```

Comment:
also add kill_vm = yes in case we hit an issue before we stop vm

Reply:
Have update, thanks zhenchao.